lib/gui/models/selection-state.js

```diff
@@ -16,338 +16,325 @@
 
 'use strict';
 
-/**
- * @module Etcher.Models.SelectionState
- */
-
 const _ = require('lodash');
-const angular = require('angular');
 const Store = require('./store');
 const availableDrives = require('./available-drives');
-const MODULE_NAME = 'Etcher.Models.SelectionState';
-const SelectionStateModel = angular.module(MODULE_NAME, []);
 
-SelectionStateModel.service('SelectionStateModel', function() {
+/**
+ * @summary Set a drive
+ * @function
+ * @public
+ *
+ * @param {String} drive - drive device
+ *
+ * @example
+ * selectionState.setDrive('/dev/disk2');
+ */
+module.exports.setDrive = (drive) => {
```

Comment:
Let's use just `exports.<function>`, since the initial `module` is redundant (CommonJS automatically aliases `exports` to `module.exports`).